manifests/dev/rbac.authorization.k8s.yaml.in

```diff
@@ -229,3 +229,83 @@ subjects:
   - kind: ServiceAccount
     name: kubevirt-privileged
     namespace: {{.Namespace}}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kubevirt.io:admin
+  labels:
+    kubevirt.io: ""
+rules:
+  - apiGroups:
+      - subresources.kubevirt.io
+    resources:
+      - virtualmachines/console
+      - virtualmachines/vnc
+    verbs:
+      - get
+  - apiGroups:
+      - kubevirt.io
+    resources:
+      - virtualmachines
+      - offlinevirtualmachines
+      - virtualmachinepresets
+      - virtualmachinereplicasets
+    verbs:
+      - get
+      - delete
+      - create
+      - update
+      - patch
+      - list
+      - watch
+      - deletecollection
```

Comment:
Is there a reason why `deletecollection` is only allowed to admins?

Reply:
my thought process is that deletecollection is a high risk endpoint that isn't necessary or used under normal operation.

For example, If I was running an operations team, I'd hand out the ability to edit VMs to everyone on my team. I wouldn't want anyone (including myself) to have the ability to accidentally wipe out all vms across all namespaces. 

It's still possible to run a command to list all vms and then delete each individual one, but that requires more deliberate forethought.

Reply:
If that is also what we can expect per default in e.g. OpenShift when a new namespace is created, it works for me.

Reply:
We can change it anway any time. :+1: